### PR TITLE
Fix reflection for one line docblocks

### PR DESCRIPTION
**Pull request by lstrojny**
This fixes an issue where the parsing of single line docblocks fail and results in a syntax error with Ocramius/ProxyManager.

/cc @Ocramius

### CODE REVIEW

**Comment by malukenho** [comment]
@lstrojny build is failing

File: tests/ZendTest/Code/Reflection/TestAsset/TestSampleClass13.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace ZendTest\Code\Reflection\TestAsset;
+
+/** Short Class Description */
+class TestSampleClass13
+{
+    /** Short Method Description */
+    public function doSomething()
+    {}
```
Comment (malukenho) [inline tests/ZendTest/Code/Reflection/TestAsset/TestSampleClass13.php]: You can put each curly braces on one line.

Comment (weierophinney) [inline tests/ZendTest/Code/Reflection/TestAsset/TestSampleClass13.php]: In this case, it makes more sense to omit this class from our php_cs rules.

Comment (weierophinney) [inline tests/ZendTest/Code/Reflection/TestAsset/TestSampleClass13.php]: Actually, that path is already excluded from our php-cs-fixer ruleset.

**Comment by weierophinney** [comment]
@malukenho Note: please do not comment about build failures; we have status integration already with Travis, and authors will get notifications of build failures. Currently, *all* builds against master fail due to an environment issue anyways, so we need to check the results manually anyways to see if new, unknown failures are present.

File: tests/ZendTest/Code/Reflection/TestAsset/TestSampleClass13.php (hunk 1 above)
Comment (Ocramius) [inline tests/ZendTest/Code/Reflection/TestAsset/TestSampleClass13.php]: This asset is actually supposed to have this particular structure to trigger a failure

**Comment by malukenho** [comment]
@weierophinney Okay, not problem!

**Comment by weierophinney** [comment]
Merged to develop for release with 2.4.

**Pull request closed without merging**